site/includes/components/donation_buttons.njk

```diff
@@ -6,4 +6,6 @@
     <div class="py-2 md:px-2 flex justify-center">
         <a href="/hospitals" class="bg-gray-200 hover:bg-gray-300 transition-all ease-in-out duration-300 py-3 px-5 md:py-4 md:px-6 rounded cursor-pointer text-blue-800 text-xl">We're a hospital</a>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+{% include "components/social-share.njk" %}
```

Comment:
I think if you move this out of this partial then it won't show up below the top buttons (where it might be a bit overwhelming). Then you can put it below the donation form in that partial too?

Reply:
You'll also have to put it in the footer partial manually too.